code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -44,6 +44,7 @@
 	move_to_delay = 6
 	speed = 3
 	mob_size = 6
+	smart_melee = FALSE
```

Comment:
This would disable the smart melee for all the gremorians, since this is the base class

Reply:
damn............................

Reply:
fixed it